README.md

```diff
@@ -45,7 +45,8 @@ Currently, supported tools include:
 |[FastQ Screen][fastq-screen]     |[Tophat][tophat]        |[SnpEff][snpeff]           |[QUAST][quast]|
 |[Flexbar][flexbar]               ||[Subread featureCounts][featurecounts]             |[RNA-SeQC][rna_seqc]|
 |[Jellyfish][jellyfish]           |                        |[RSEM][rsem]               |[RSeQC][rseqc]|
-|[Skewer][skewer]                 |                        |[THetA2][theta2]           |[Sargasso][sargasso]|
+|[KAT][kat]                       |                        |[THetA2][theta2]           ||
```

Comment:
`[kat]` needs a hyperlink reference to the currently non-existent MultiQC docs page (will be automatically built when the PR is merged). See at the bottom of the readme file for all of the others.